_includes/hire_us/new_projects.html

```diff
@@ -0,0 +1,27 @@
+<!-- Hire Us: New Projects -->
+<section id="new-projects" class="hire-us--section-new-project">
+    <div class="container">
+        <div class="row">
+            <div class="col text-center">
+                <h2 class="section-heading">For new projects</h2>
+            </div>
+        </div>
+        <div class="row justify-content-center mb-5">
+            <div class="col-md-7 text-center">
+                <h3 class="section-subheading text-muted">For projects where we staff all developer roles, we like to work as follows:</h3>
+            </div>
+        </div>
+        <div class="row text-center">
+            <div class="col">
+                <h4 class="service-heading">Scrum</h4>
+                <p class="text-muted">We carry out our work in two-week Sprints. We ensure that four key roles are
+                    present on the team: Product Owner, Scrum Master, Developers and QA Engineer(s).</p>
```

Comment:
s/present/covered/ ?
The way QA Eng is worded might send a different message than what was intended.

Reply:
Will leave this feedback for @calumhalcrow, as I just copy/pasted his content from "custom software development" service.